pkgs/tools/networking/strongswan/default.nix

```diff
@@ -40,19 +40,6 @@ stdenv.mkDerivation rec {
     ./ext_auth-path.patch
     ./firewall_defaults.patch
     ./updown-path.patch
-
-    # Don't use etc/dbus-1/system.d
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
Damnit...